docs/docs/projects/add-project.md

```diff
@@ -16,4 +16,5 @@ links:
 1. Provide a name to the `Project` being added. For example, "My Product"
 1. Click `CONFIRM`
   - You will be redirected to the `Project resources` page, with details of your newly created project
-  - You can now manage `Environments`, `Teams` and `Roles` in your new project
+  - You can now manage [/docs/environments/index], [/docs/teams/index] and [/docs/roles/index] in your new project
+  <!-- - You can now manage `Environments`, `Teams` and `Roles` in your new project -->
```

Comment:
@Now-Tiger maybe use the [Markdown Links syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links) here?

```suggestion
  - You can now manage [Environments](/docs/environments/index), [Teams](/docs/teams/index) and [Roles](/docs/roles/index) in your new project
```